package.json

```diff
@@ -18,8 +18,10 @@
     "test": "./node_modules/.bin/grunt && ./bin/jasmine.js"
   },
   "dependencies": {
-    "glob": "^7.1.3",
-    "jasmine-core": "~3.3.0"
+    "es6-object-assign": "^1.1.0",
```

Comment:
I'm fine with switching from glob to fast-glob since the interface looks better, but I don't want to add the other two runtime dependencies (es6-object-assign, and lodash.noop). We want to keep Jasmine's dependency graph small. If `Object.assign` doesn't exist in Node.js v4, we'll just have to find a different way to do that. I think jasmine-core has an `extend` function in boot that it uses for the same thing.

Reply:
Node.js v4 has Object.assign. See https://node.green/#ES2015-built-in-extensions-Object-static-methods-Object-assign